dependent_type_theory.md

```diff
@@ -322,7 +337,7 @@ def g (s : String) : Bool := s.length > 0
 
 Think about what these expressions mean. The expression
 ``fun x : Nat => x`` denotes the identity function on ``Nat``, the
-expression ``fun x : Nat => true`` denotes the constant function that
```

Comment:
I think we should keep the "constant" here. It is a constant function.

Reply:
fixed